system-addon/content-src/components/Search/_Search.scss

```diff
@@ -60,7 +64,17 @@
 
     &:focus,
     &:hover {
-      background-color: $grey-90-10;
+      &::before {
+        content: "";
+        position: absolute;
+        top: 0;
+        offset-inline-start: 0;
+        width: 100%;
+        height: 100%;
+        border-radius: 0 $border-radius $border-radius 0;
```

Comment:
nit: Let's share this inside `.search-button` (?) as a variable

Reply:
Or.. if we have a slightly smaller radius, will the whites in the corner be filled in? !

Reply:
Oh indeed! With 2px. ! ? !
![screen shot 2017-11-01 at 2 44 05 pm](https://user-images.githubusercontent.com/438537/32299520-2a920816-bf13-11e7-8970-4dc87a989209.png)